README.md

```diff
@@ -1,34 +1,40 @@
 [![PyPI version](https://badge.fury.io/py/django-admin-autocomplete-filter.svg?kill_cache=1)](https://badge.fury.io/py/django-admin-autocomplete-filter)
 
+
 Django Admin Autocomplete Filter
 ================================
-A simple Django app to render list filters in django admin using autocomplete widget. This app is heavily inspired from [dal-admin-filters.](https://github.com/shamanu4/dal_admin_filters)
+A simple Django app to render list filters in django admin using an autocomplete widget. This app is heavily inspired by [dal-admin-filters.](https://github.com/shamanu4/dal_admin_filters)
+
 
 Overview:
 ---------
 
 Django comes preshipped with an admin panel which is a great utility to create quick CRUD's.
```

Comment:
Can we improve the first line too? :slightly_smiling_face: 
- plural forms are not marked with apostrophe
- CRUD as a noun is weird

> Django comes installed with an admin panel that is a great utility to create a quick content editing interface.

Reply:
If you'd like to improve a different part of the text, why not issue your own PR?